app/models/header_lookup.rb

```diff
@@ -106,13 +106,13 @@ def lcsh_term_pid_lookup(lcsh_term="")
   end
 
   def lcnaf_pid_lookup(lcnaf_term)
-    lcnaf_term = lcnaf_term.downcase.trim
+    lcnaf_term = lcnaf_term.downcase.strip
```

Comment:
I think this was something I changed in-line while building the memoized LCNAF file, but not something I changed in version control. Whoops.